install_config/install/prerequisites.adoc

```diff
@@ -806,6 +806,51 @@ See
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Logical_Volume_Manager_Administration/index.html[Logical
 Volume Manager Administration] for more detailed information on LVM management.
 
+[[managing-docker-container-logs]]
+*Managing Docker Container Logs*
+
+Sometimes a container's log file
+(*_/var/lib/docker/containers/HASH/HASH-json.log_*) can increase to a
+problematic size. You can manage this by configuring the `json-file` logging
+driver to restrict the size and number of log files.
+
+For example, `--log-opt max-size` rolls over the size, and you can specify the
+`--log-opt max-file`, which is the maximum number of files that a log is rolled
+over before being discarded. You can use these options with Docker 1.8.2 or
+later.
+
+Edit the *_/etc/sysconfig/docker_* file to configure `max-size=1M` and
+`max-file=3`:
+====
+----
+OPTIONS='--insecure-registry=172.30.0.0/16 --selinux-enabled --log-opt max-size=1M --log-opt max-file=3' <1>
+----
+====
+<1> `json.log` rollover by 1MB and it stored 3 files.
+
+Next, restart the Docker service:
+----
+# systemctl restart docker.service
+----
+
+Log output example:
```

Comment:
Probably rewrite this into a sentence or make it a title for the following example block, i.e. add a `.Viewing Available Container Logs` or something on the line right before the example block.